anteform.p8

```diff
@@ -1213,7 +1166,7 @@ end
 -- drawing routines
 
 function delay(numofcycles)
-  for delaycount=0,numofcycles do
+  for delaycount=1,numofcycles do
     flip()
   end
 end
```

Comment:
Was it intentional that the previous code would wait two frames for numofcycles==1, three frames for 2, etc?